docs/read-from-object-store-layer.md

```diff
@@ -0,0 +1,141 @@
+# Work with an object store layer
```

Comment:
The file name "read-from-object-store-layer.md" is a bit misleading when this also covers write?

Reply:
good catch, will rename. thanks